webapp/src/components/Sidebar.js

```diff
@@ -57,48 +58,49 @@ const useStyles = makeStyles((theme) => ({
 
 const DashboardSidebarContent = () => {
   const classes = useStyles()
+  const { t } = useTranslation('sidebar')
 
   const mainPages = [
     {
-      title: 'Block Producers',
+      title: t('item1'),
```

Comment:
I think that could be better use "BLOCK_PRODUCER_MENU_OPTION" or something like that